# retrofit
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -u '/home/jonathan/Code/lowarn/examples/git/versions/1000/simplified/xmonad.cabal' '/home/jonathan/Code/lowarn/examples/git/versions/1000/retrofitted/xmonad.cabal'
Index: ./xmonad.cabal
--- ./xmonad.cabal	Thu Jan 01 00:00:00 1970
+++ ./xmonad.cabal	Thu Jan 01 00:00:00 1970
@@ -1,4 +1,4 @@
-name:               xmonad-lowarn
+name:               lowarn-xmonad-lowarn
 version:            0.7
 homepage:           http://xmonad.org
 synopsis:           A tiling window manager
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Thu Jan 01 00:00:00 1970
# Generated by        : makepatch 2.03
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p 'xmonad.cabal' 2700 1681299481 0100644
#### End of ApplyPatch data ####

#### End of Patch kit [created: Thu Jan 01 00:00:00 1970] ####
